docs/index.rst

```diff
@@ -0,0 +1,15 @@
+Documentation
+=============
+
+This is the documentation for pyllisto.
+
+
+.. toctree::
+  :maxdepth: 2
+
+  pyllisto/index.rst
+
+.. note:: The layout of this directory is simply a suggestion.  To follow
+          traditional practice, do *not* edit this page, but instead place
+          all documentation for the package inside ``pyllisto/``.
+          You can follow this practice or choose your own layout.
```

Comment:
I think you can modify this to put the main index for the docs in ``docs/index.rst``